src/js/components/Crumb.js

```diff
@@ -0,0 +1,28 @@
+import {Link} from 'react-router';
+import React, {PropTypes} from 'react';
+
+class Crumb extends React.Component {
```

Comment:
This component will allow us to create a single Crumb, that is then later wrapped in an `li` at least for now.